qiskit_experiments/library/randomized_benchmarking/interleaved_rb_experiment.py

```diff
@@ -98,7 +138,14 @@ def __init__(
             seed=seed,
             full_sampling=full_sampling,
         )
-        self._transpiled_interleaved_elem = None
+        # Convert interleaved element to integer for speed
+        if self.num_qubits <= 2:
+            interleaved_circ = self._interleaved_elem.to_circuit()
+            utils = CliffordUtils(
+                self.num_qubits, basis_gates=self._get_basis_gates()
+            )  # TODO: cleanup
+            self._interleaved_elem = utils.compose_num_with_clifford(0, interleaved_circ)
```

Comment:
`_interleaved_elem` may not be an integer since its type is `SequenceElementType = Union[Clifford, Integral, QuantumCircuit]`. So I believe it's not misleading.